test/common/services/security/test_user.ts

```diff
@@ -81,7 +81,7 @@ export async function createTestUserService(
         });
 
         if (browser && testSubjects) {
-          if (await testSubjects.exists('kibanaChrome')) {
+          if (await testSubjects.exists('kibanaChrome', { timeout: 10000, allowHidden: true })) {
```

Comment:
The default timeout for `exists` is 2.5 seconds, why should this one be 10?

Reply:
If we want it to wait for longer, maybe we can do it like below where we multiply the default, but I feel like we shouldn't need to change the timeout every time we use these methods.

Reply:
in `dashboard_only_mode` kibanaChrome` is hidden, but deaults to displayed in regular mode. removed the timeouts and added `allowHidden: true `